sections/02_intro.md

```diff
@@ -34,41 +34,88 @@ learning methods more challenging or less fruitful to apply.
+
+Several important advances make the current surge of work done in this area possible.
+
+First, several easy to use software packages (Tensorflow, Caffe, Theano) now enable a much broader range of scientists
+to build and train complicated. In the past, neural networks required very specialized knoweldge to 
+build and modify, including the ability to robustly code differentials of matrix 
+expressions. Errors here are often subtle and difficult to detect, so it could be 
+very difffult to tailor networks to specific problems without substantial experience
+and training. Now, however, with these new packages, even very complex neural networks 
+are automatically differentiated, and high level scripting instructions can transparently
+run very efficently on GPUs. The technology has progressed to the point that even
+algorithms can be differentiated [cite neural stack and neural memory papers].
+
+Second, key technical insight has been uncovered that guides the construction of 
+much more complicated networks that previously possible. In the past, most neural networks 
+included just a single network. A network with an arbitrary number of hidden nodes, but 
+just a single layer, can learn arbitrarily complex functions. And networks with more than one hidden
+layer (deep networks), were hard to train. However, it turns out, deep networks can more
+efficiently represent many tasks when they are built to mirror the underlying structure of the data.
+Moroever, deep networks are more robust and trainable when employing several
+architectural innovations: weight replication, better behaived non-linearities like rectified-linear units, residual networks, 
+and better weight initialization, and persistent memory. Likewise the central role of 
+dimensionality reduction as a strength of neural networks was elucidated, and this has
+motivated designs built to capitalize on this strength [cite autoencoders and word2vec].
+
+Third, several advances in training algorithms have enabled applications of neural networks in ways
+not obviously possible. The number of training strategies for deep learning is growing rapidly and a complete
+reveiw is beyond our scope. But these algorithms can train networks in domains where earlier algorithms struggled. 
+For example, newer optimizers can very efficiently learn using batched training, where only a portion of the data
+needs to be processed at a time. These optimizers more effectively optimize very large weight vectors where many weights are only
+rarely updated. Noise constrastive error has proven particularly useful in 
+modeling language. Reinforcement learning has enabled neural networks to learn how to play games 
+like chess, GO, and poker. Curriculumn learning enables networks to gradually build up expertise to
+solve particularly challenging algorithmic problems. Dropout nodes and layers make networks much more
+robust, even when the number of weights are dramatically increased.
+
```

Comment:
There is quite a bit of jargon in here. If it's going to be included, citations need to exist to point readers towards a resource for each topic.